packages/app/test/storeBuilders.ts

```diff
@@ -589,9 +589,7 @@ export const buildStoreState = () => {
             completion: 1,
             track: {
               uuid: '1',
-              artist: {
-                name: 'test artist 1'
-              },
+              artist: 'test artist 1',
```

Comment:
I think the format here should stay as it was, since artist can be either a string or an object

Reply:
This is true for the Track type used commonly across the app. But before the track is saved to the store, it is mapped to the TrackItem type (See getTrackItem) which I have typed. The full type for the object stored in state is the new type Download.

Reply:
Ok, I just realized that since the tests pass this works either way.